openapi/components/schemas/Invoices/InvoiceItem.yaml

```
@@ -65,7 +65,7 @@ properties:
     type: string
     format: date-time
   periodNumber:
-    description: Number of subscription order billing periods in which the invoice item has been included.
+    description: Consecutive number of the billing period which is covered by the invoice item.
```

nitpick: I don't understand want is meant by 'covered by'

I've made some suggestion here, please feel free to edit them.

```suggestion
    description: Consecutive number of billing periods in which the invoice item is included.
```

```suggestion
    description: Consecutive number of billing periods where the invoice item is added.
```

---

I feel they are both off. 
- A subscription has consecutive billing periods: №1, №2, №3 etc, each being paid by invoice (with items).
- The invoice item pays e.g. shaving service for billing period №3, but it might've been absent at billing period №2 for instance. The only fact is it relates to billing period №3.

English "number" reflects both quantity and consecutive index numbers, hence the confusion, I guess.